packages/waku/src/lib/renderers/rsc-renderer.ts

```diff
@@ -169,11 +169,15 @@ export async function renderRsc(
         elementsPromise = Promise.all([
           elementsPromise,
           renderEntries(rscPath, { rscParams }),
-        ]).then(([oldElements, newElements]) => ({
-          ...oldElements,
-          // FIXME we should actually check if newElements is null and send an error
```

Comment:
I don't remember when it becomes `null`. Have you tried anything?

Reply:
When the path is not found: https://github.com/dai-shi/waku/blob/main/packages/waku/src/router/define-router.ts#L126

Maybe we could throw the error from `renderEntries` when path is not found instead of returning null? That way the path could be included in the error message.

Reply:
Hmm, I actually don't remember why I wrote this comment, particularly "send an error". Let's make it `console.warn` instead of throwing an error for now.

Reply:
I can change to `console.warn` but I should say: this would throw a runtime error if `null` is returned with the current version on `main` because null cannot be spread `...null` or `...someVariableThatIsNull` both throw runtime errors.

So given that we haven't seen this reported as an issue people run into, it seems like `null` is never actually getting returned which is good.

Reply:
`{ ...null }` is fine. at least, works on nodejs.

Reply:
weird... errors in the browser

<img width="399" alt="image" src="https://github.com/user-attachments/assets/8a0c79ee-c803-4566-8a34-b0586cbe1165">

I'll change to console.warn though!

Reply:
oh... but, yeah.